editor/components/block-settings-menu/index.js

```diff
@@ -21,8 +21,8 @@ import BlockDuplicateButton from './block-duplicate-button';
 import BlockRemoveButton from './block-remove-button';
 import SharedBlockConvertButton from './shared-block-convert-button';
 import SharedBlockDeleteButton from './shared-block-delete-button';
-import UnknownConverter from './unknown-converter';
-import HTMLConverter from './html-converter';
+import ConvertToBlocksFromHTMLButton from './convertToBlocksFromHTML';
+import ConvertToBlocksFromUnknownButton from './convertToBlocksFromUnknown';
```

Comment:
`ConvertToBlocksFromUnknownButton` - it would be nice to find a different name. At the moment the colocation of `unknown` and `button` is a bit unfortunate :(

`HtmlToBlocksConvertButton` - how about this?

Reply:
https://github.com/WordPress/gutenberg/blob/4263e556fc889206fee5b22d54a1cd910f4e586b/docs/reference/coding-guidelines.md#abbreviations-and-acronyms

So it would be: `HTMLToBlocksConvertButton` and `UnknownToBlocksConvertButton`.

Reply:
I'd like to find a name that collocates the three components together, like `SharedBlockConvertButton` and `SharedBlockConvertButton` convey that they act on shared blocks.

Not sure how to improve the current ones :thinking:

Reply:
How about `BlockConverterFromHTMLButton` and `BlockConverterFromUnknownButton`? That way we follow the current naming schema (other files either start by block or shared-block).

Reply:
I appreciate the time you've invested to share that observation and to help me figure out this!

In 0327937 I decided for `BlockUnknownConvertButton` and `BlockHTMLConvertButton` which addresses the concerns and is more in line with our current naming schema. We can iterate later if necessary.